src/Orleans/Serialization/OrleansJsonSerializer.cs

```diff
@@ -61,10 +64,13 @@ public static JsonSerializerSettings UpdateSerializerSettings(JsonSerializerSett
             if (config.Properties.ContainsKey(UseFullAssemblyNamesProperty))
             {
                 bool useFullAssemblyNames;
+#if !NETSTANDARD_TODO
+                //Newtonsoft.Json.JsonSerializerSettings.set_TypeNameAssemblyFormat not suppported in netstanard
                 if (bool.TryParse(config.Properties[UseFullAssemblyNamesProperty], out useFullAssemblyNames) && useFullAssemblyNames)
                 {
                     settings.TypeNameAssemblyFormat = FormatterAssemblyStyle.Full;
                 }
+#endif
```

Comment:
it was weird for me since the method builds well in compile, but cannot be loaded in runtime. 

Newtonsoft.Json suggest to use TypeNameAssemblyFormatHandling. since TypeNameAssemblyFormat is obsolete
see
https://github.com/JamesNK/Newtonsoft.Json/blob/17f87f53211af9645ab8b6947d79e9f4e0e3c6be/Src/Newtonsoft.Json/JsonSerializerSettings.cs#L185

Will take a closer look on Monday. Just tried to use it in vNext, compiler cannot find TypeNameAssemblyFormatHandling method...